clients/privacy-center/pages/consent.tsx

```diff
@@ -26,10 +26,10 @@ const Consent: NextPage = () => {
   const router = useRouter();
 
   useEffect(() => {
-    if (!consentRequestId || !verificationCode) {
+    if (!consentRequestId) {
       router.push("/");
     }
-  }, [consentRequestId, verificationCode, router]);
+  }, [consentRequestId, router]);
```

Comment:
I don't know the expected flow of this too well, but I wonder if there is a place where we need to clear out stale values? 

or perhaps instead of local storage, we should use query parameters? though that would only work if this page and the home page are the only places that use these codes/ids. probably out of the scope of this PR tho

Reply:
There probably is a better way to do this. I think the privacy center will need a refactor to use RTK query at some in the near future. Maybe those values could live in the redux store?